src/Spinner/Spinner.js

```diff
@@ -243,5 +246,13 @@ module.exports = kind(
 	* @see enyo/AccessibilitySupport~AccessibilitySupport#accessibilityLive
 	* @public
 	*/
-	accessibilityLive: 'off'
+	accessibilityLive: 'off',
+
+	/**
+	* @default Loading
```

Comment:
Public property should have well documented comment. I think this document should also include sample code for how do you expect app developer to use this constant.